Dockerfile

```diff
@@ -99,23 +99,30 @@ ENV NPM_DEBUG=true
 
 RUN \
     # Files needed to run the make command
-    --mount=type=bind,source=Makefile,target=${HOME}/Makefile \
     --mount=type=bind,source=Makefile-docker,target=${HOME}/Makefile-docker \
     # Files required to install pip dependencies
-    --mount=type=bind,source=setup.py,target=${HOME}/setup.py \
-    --mount=type=bind,source=./requirements,target=${HOME}/requirements \
+    --mount=type=bind,source=./requirements/pip.txt,target=${HOME}/requirements/pip.txt \
+    --mount=type=cache,target=${PIP_CACHE_DIR},uid=${OLYMPIA_UID},gid=${OLYMPIA_UID} \
+    # Command to install dependencies
+    make -f Makefile-docker update_deps_pip
```

Comment:
All we need in "base" is pip to be installed, this let's other layers install packages using the correct version.